libs/ui/src/lib/detailed-status/detailed-status.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  ViewChild
+} from '@angular/core';
+import {
+  animate,
+  state,
+  style,
+  transition,
+  trigger
+} from '@angular/animations';
+import { TerminalComponent } from '../terminal/terminal.component';
+import { CommandResponse } from '@angular-console/utils';
+
+const ANIMATION_DURATION = 250;
+const COMMAND_REGEXP = /^\w+\s+([a-zA-Z0-9-_]+?)(\s+.*)?$/;
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'ui-detailed-status',
+  templateUrl: './detailed-status.component.html',
+  styleUrls: ['./detailed-status.component.scss'],
+  animations: [
+    trigger('growShrink', [
+      state('void', style({ width: '100vw' })),
```

Comment:
When we switch to a more desktop friendly layout its likely that using 100vw/30vw won't work the way we want it to. Could you use '*' or a percentage instead?